skimage/typing/_image_types.py

```diff
@@ -0,0 +1,14 @@
+from ._array_like import ArrayLike
+import numpy as np
+
+from typing_extensions import Annotated
```

Comment:
I would expect this to crash with py3.6 but apparently it does not ?

Reply:
(a) that's fascinating, and (b) who cares, apparently today is Py3.7 day! =D

Reply:
well we have py3.6 builds but they pass (not the ones building the docs but this is not an import problem)

Reply:
what I'm saying is we can drop those builds so this is not really worth worrying about. =) See: https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table

Reply:
If we drop the builds we accept that *de facto* future versions will not support py3.6. Can't believe that Google Colab is still supporting only py3.6 but I just checked and it seems to be the case. So we can make this decision but it should be motivated by more than just this PR.

Reply:
It's motivated by following NEP29. I'm super not interested in supporting Google Colab or other proprietary technologies.

Reply:
I totally understand why you're not interested :-). Nevertheless I think a lot of people are using Colab for deep learning on images and scikit-image for pre/post-processing.